errbot/backends/test.py

```diff
@@ -359,85 +345,10 @@ def tearDown(self):
 
     def assertCommand(self, command, response, timeout=5):
         """Assert the given command returns the given response"""
-        push_message(command)
-        self.assertIn(response, popMessage(), timeout)
+        self.bot.push_message(command)
+        self.assertIn(response, self.bot.pop_message(), timeout)
 
     def assertCommandFound(self, command, timeout=5):
         """Assert the given command does not exist"""
-        push_message(command)
-        self.assertNotIn('not found', popMessage(), timeout)
-
-
-@pytest.fixture
-def testbot(request):
```

Comment:
It looks like this is removed here but not re-introduced anywhere else, but I would like to keep it so plugin authors can continue to write tests like this:

```python
        # myplugin_test.py
        from errbot.backends.test import testbot, push_message, pop_message

        def test_about(testbot):
            push_message('!about')
            assert "Err version" in pop_message()
```